### PR TITLE
implemented add files method in LLMRails

### PR DESCRIPTION
**Pull request by anar2706**
This PR provides add files method with LLMRails. Implemented here are:

docs/extras/integrations/vectorstores/llm-rails.ipynb

### CODE REVIEW

File: libs/langchain/langchain/vectorstores/llm_rails.py [hunk 1]
```diff
@@ -47,14 +41,11 @@ def __init__(
 
         self._session = requests.Session()  # to reuse connections
         self.datastore_id = datastore_id
-        self.base_url = "https://api.llmrails.com/v1"
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/llm_rails.py:50]: should this be user configurable?

Comment (anar2706) [reply libs/langchain/langchain/vectorstores/llm_rails.py:50]: No it should not missed it when developing fixing now

**Comment by anar2706** [comment]
@baskaryan  fixed it.

**Comment by baskaryan** [comment]
lgtm, thanks @anar2706!

**Pull request merged**